scheduler/src/cook/mesos/api.clj

```diff
@@ -1108,7 +1116,7 @@
                        (catch Exception e
                          (log/warn e "Malformed raw api request")
                          [true {::error (.getMessage e)}]))))
-
+     :allowed? (partial job-create-allowed? is-authorized-fn)
```

Comment:
Is it possible to add an integration test where the creation of a job is disallowed?